### DISCUSSION TITLE
Expressing

### DISCUSSION THREAD

**Discussion by TaddyJ** [babybuddy/babybuddy#364] [Ideas]
My wife and I had a premmie baby. 
Unfortunately he was unable and to some degree still is unable to breast feed so my wife has been expressing.

One thing that would be nice is another section for expressing.
Something fairly simple like the + feeding section would be easy. 
Sections like;

Time: Start/Finish
Method: Single/Double 
> Single - Side/Amount:
> Double - Left Amount
> Double - Right Amount
Notes:

And the graphing section for for the diaper changes would be nice for the double to track the amounts from each side since they tend to never be the same.

**Comment by cdubz** [comment]
A similar request is #159 and ultimately we're trying to think about making the capture of other things more general in #217. That said, I can see why tracking this separately would be ideal.

**Reply by TaddyJ** [reply]
Ahh. Did not see those separately. Thanks for the response. :)
